Pinta.Core/Actions/EditActions.cs

```diff
@@ -374,6 +374,14 @@ private void HandlerPintaCoreActionsEditLoadPaletteActivated (object sender, Eve
 				}
 			}
 
+			if (SystemManager.GetOperatingSystem() != OS.Windows) {
```

Comment:
Also just noticed this was the open palette dialog :) This should be moved over to `LayerActions.cs` for the Import from File action